lerna.json

```diff
@@ -1,8 +1,7 @@
 {
   "packages": [
     "packages/**/*",
-    "sites/fast-color-explorer",
-    "sites/fast-website"
+    "sites/fast-color-explorer"
```

Comment:
This breaks the `prebuild` step for `sites/website`.

Reply:
Hmm. If I add it back will it break the package install for the homepage or do you know if there's some other fix? I haven't worked in the home page's code before so I was sort of playing whackamole a little bit to try to get it working.

Reply:
Structurally, the homepage is considered its own package, which we build before running Docusaurus. Its `dist` files are then copied to Docusaurus' `static` directory, so that Docusaurus picks it up and includes it as part of its bundle.